### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by OtiKelsy7** [Programmers-Hub-International/manage-landing-page-master#6] [Announcements]
Please what's the update for the mobile view version devs

**Comment by Walter-Tronics** [comment]
Mention everyone and let them comment or highlight themselves, those in the mobile view. 
@TheAce74 and 
@Syipmong are gonna be leading the team of mobile. 
If everyone has indicated, @TheAce74 could start assigning sections to each of person

**Reply by TheAce74** [reply]
Noted boss 🔥

**Comment by Syipmong** [comment]
Noted with thanks sir

**Comment by OtiKelsy7** [comment]
So how's it going sirs

**Reply by Walter-Tronics** [reply]
This is not playing out well. Everyone has to comment on their status so the stuff can commence

**Comment by TheAce74** [comment]
Like our boss @Walter-Tronics stated, we haven't made any progress in the mobile view cus we've been kind of wishy-washy. I think I have an idea on what to do if you guys are up for it.

I think three people worked on the desktop view
@Walter-Tronics 
@TomiwaAborowa 
@wizzy-design 

So I'll split the components I mentioned earlier between the remaining names. I'm currently working on a project so I'll start my part from tomorrow if the ones before me finish theirs. We'll follow this order in working on the mobile view:
 
**Header**
@Einsteinj04 
@Syipmong 

**Hero**
@kingchi005 
@OtiKelsy7

**About**
@Paulcode405
@devnasfam

**Testimonial**
@marviecephas 

**CTA and footer**
@TheAce74 

If you agree, then you can start working immediately 
Thank you all ✌️

**Reply by Walter-Tronics** [reply]
Fully in support. Please everyone get on your aspect. 
@TheAce74 I will create issues and assign them assignees based on your dissection.

**Reply by OtiKelsy7** [reply]
Odogwu add me again 😫

**Reply by Walter-Tronics** [reply]
@OtiKelsy7 add you where??

**Reply by OtiKelsy7** [reply]
Never mind 😌

**Reply by TheAce74** [reply]
> Fully in support. Please everyone get on your aspect. 
> @TheAce74 I will create issues and assign them assignees based on your dissection.

Ok my boss

**Comment by Walter-Tronics** [comment]
If you have started your section, simply go to the project and move your to-do task to the "In progress" place so it's in order.
@TheAce74 
@TomiwaAborowa 
@wizzy-design 
@devnasfam 
@Einsteinj04 
@Syipmong 
@kingchi005 
@OtiKelsy7 
@Paulcode405 
@marviecephas

**Reply by wizzy-design** [reply]
Why am i tagged? Am i to contribute to mobile too?

**Reply by Walter-Tronics** [reply]
No, I just tagged everyone so we all know what I'm talking about.

**Comment by Syipmong** [comment]
Will do sir
Thanks

**Comment by Syipmong** [comment]
Maybe but please don't take my assigned work o😅👍🏻

**Comment by Syipmong** [comment]
We cool as long as you don't touch my portion😂😂😂😂

**Comment by marviecephas** [comment]
Hey nerds! 🤓

Please if you observed, I haven't really been active. I don't have a pc and
the little phone I use to code got spoilt 😭 I really want to be part of
this project but unfortunately 😔...... Had to login my email on a friend's
device to send this... See you guys when I'm whole ❤️.

Ikwerre Dev
marvie.

**Reply by Walter-Tronics** [reply]
I pray it becomes well with you 🙏

**Comment by TheAce74** [comment]
Please what's been happening here? Since we agreed on this stuff, no one has started

**Reply by Walter-Tronics** [reply]
I really don't know. They're not taking this seriously. Maybe they're not interested after all.

@TheAce74 
@TomiwaAborowa 
@wizzy-design 
@devnasfam 
@Einsteinj04 
@Syipmong 
@kingchi005 
@OtiKelsy7 
@Paulcode405 
@marviecephas

**Reply by OtiKelsy7** [reply]
I'm sorry but I've been busy with school and my partner hasn't been replying me

**Reply by Walter-Tronics** [reply]
Okay

**Comment by Syipmong** [comment]
Sir why didn't you merge my branch before the latest edit


Shah no p I'll continue

**Reply by Walter-Tronics** [reply]
Did you make a pull request??
I can't find it

**Reply by Walter-Tronics** [reply]
You created a new branch! 
I have seen your changes, but you're not yet done with it ( header and nav ). When you're done, I will merge it into the main branch.

**Comment by Syipmong** [comment]
Ok boss
